RiotSharp/Misc/DateTimeConverterFromStringTimestamp.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace RiotSharp
+{
+    class DateTimeConverterFromStringTimestamp : JsonConverter
```

Comment:
Yes. Riot sends the DateTime as a timestamp encoded in a string. Don't ask me why.